movies-service/src/main/java/com/pentalog/pentastagiu/repository/model/actors/Actor.java

```diff
@@ -15,25 +19,98 @@
     private String id;
     @NotEmpty(message = "Movie name must be provided!")
     private String name;
+    @Min(value = 0, message = "Number of Oscars cannot be lower than 0!")
+    @Max(value = 200, message = "Number of Oscars cannot be greater than 200!")
+    @NotNull(message = "You need to specify a number!")
+    private Integer numberOfOscars;
+    @NotEmpty(message = "Facebook account url must not be empty!")
+    private String facebookAccountUrl;
+
+    @ManyToMany(fetch = FetchType.LAZY, mappedBy = "actors", cascade = CascadeType.ALL)
+    private Set<Movie> movies = new HashSet<>();
 
     public Actor() {
     }
 
+    public Actor(String id, @NotEmpty(message = "Movie name must be provided!") String name,
+                 @Min(value = 0, message = "Number of Oscars cannot be lower than 0!")
+                 @Max(value = 200, message = "Number of Oscars cannot be greater than 200!")
+                 @NotNull(message = "You need to specify a number!") Integer numberOfOscars,
+                 @NotEmpty(message = "Facebook account url must not be empty!") String facebookAccountUrl) {
+        this.id = id;
+        this.name = name;
+        this.numberOfOscars = numberOfOscars;
+        this.facebookAccountUrl = facebookAccountUrl;
+    }
+
+    public static class ActorBuilder {
```

Comment:
De obicei prefer sa folosesc pattern-ul de builder direct in setter, adica sa returnez in setterii de pe actor obiectul. Si asa pot inlantui setter cand fac un Actor. Metoda cu Builder nu imi place fiindca pentru fiecare clasa trebuie sa ai si o clasa de builder pe care sa o initializezi ori prin setteri ori prin constructori si apoi sa apelezi build. E prea mult overhead